tools/scripts/xabuild

```diff
@@ -160,6 +160,14 @@ function ConfigureLocalXbuild()
 	done
 }
 
+if [ -z "$CONFIGURATION" ]; then
+	for p in "$*"; do
+		case $p in
+			/property:Configuration=*|/p:Configuration=*|-p:Configuration=*|--property:Configuration=*) CONFIGURATION="`echo $p | cut -d '=' -f 2`" ;;
```

Comment:
I would prefer it if this were spread across multiple lines:

```bash
			/property:Configuration=*|/p:Configuration=*|-p:Configuration=*|--property:Configuration=*)
				CONFIGURATION="`echo $p | cut -d '=' -f 2`"
				;;
```

This makes it easier for me to read without horizontal scrolling.